### PR TITLE
Update debug.md

### PR DESCRIPTION
**Pull request by enl**
## Purpose of this pull request

Add a note that actual path to your php-fpm config can be different with given example.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/cloud/howtos/debug.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5134) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5134) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/cloud/howtos/debug.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ You can enable Xdebug directly to all Starter environments and Pro Integration e
                 start: |
                     cat /etc/php/7.0/fpm/php-fpm.conf | sed -e 's/request_terminate_timeout.*//g' > /tmp/php-fpm.conf
                     /usr/sbin/php-fpm7.0 -y /tmp/php-fpm.conf
+		    
+    Note that actual path to php-fpm can be different: check `/etc/alternatives/php` symlink on your cloud server.
```
Comment (osrecio) [inline guides/v2.2/cloud/howtos/debug.md]: I should use a info box:

```suggestion
    {: .bs-callout-info}
    Note that actual path to php-fpm can be different: check `/etc/alternatives/php` symlink on your cloud server.
```

File: guides/v2.2/cloud/howtos/debug.md [hunk 2]
```diff
@@ -62,6 +62,9 @@ You can enable Xdebug directly to all Starter environments and Pro Integration e
                 start: |
                     cat /etc/php/7.0/fpm/php-fpm.conf | sed -e 's/request_terminate_timeout.*//g' > /tmp/php-fpm.conf
                     /usr/sbin/php-fpm7.0 -y /tmp/php-fpm.conf
+		    
+    {: .bs-callout-info}
+    Note that actual path to php-fpm can be different: check `/etc/alternatives/php` symlink on your cloud server.
```
Comment (meker12) [inline guides/v2.2/cloud/howtos/debug.md]:
```suggestion
    The actual path to the `php-fpm` configuration file can be different than the one in the example. For the correct path, open an SSH connection to the Cloud environment,  and check the value for the `/etc/alternatives/php` symlink.
```
```suggestion
    The actual path to the `php-fpm` configuration file can be different than the path in the example. For the correct path, check `/etc/alternatives/php` symlink on your cloud server.
```

**Comment by osrecio** [comment]
@meker12 Do you think we can merge this PR!

**Comment by meker12** [comment]
running tests

**Pull request merged**